scylla-macros/Cargo.toml

```diff
@@ -13,3 +13,4 @@ proc-macro = true
 [dependencies]
 syn = "1.0"
 quote = "1.0" 
+proc-macro2 = "1.0"
```

Comment:
This is needed to return the Ident type in parser.rs

Reply:
Nit: add a newline at the end of file